client/signup/steps/reader-landing/index.jsx

```diff
@@ -20,7 +22,8 @@ import './style.scss';
 
 class ReaderLandingStep extends Component {
 	handleButtonClick = () => {
-		SignupActions.submitSignupStep( { stepName: this.props.stepName } );
+		this.props.recordTracksEvent( 'calypso_signup_reader_landing_cta' );
```

Comment:
Was this an intentional addition?

Reply:
Yes, here I merged the two `handleButtonClick` handlers in `ReaderLandingStep` and `ReacerLandingStepContent` into one. The inner component now just passes down a handler prop and doesn't need to be connected to Redux.